templates/topic/topic-en.html

```diff
@@ -1,95 +1,37 @@
 ---
-{
-	"title": "[Topic title]",
-	"layout": "without-h1",
-	"language": "en",
-	"altLangPage": "topic-fr.html",
-	"pageclass": "page-type-nav",
-	"breadcrumbs": [
-		{ "title": "[Theme Name]", "link": "templates/theme-en.html" }
-	],
-	"dateModified": "2023-09-01",
-	"share": "true",
-	"deptfeature": true
-}
+title: "[Topic title]"
+layout: no-container
+language: "en"
+altLangPage: "topic-fr.html"
+pageType: "topic"
```

Comment:
Do we use that page property? I think it is only useful when we are archiving a page

```suggestion
```